docs/clients/99_graphql/index.rst

```diff
@@ -36,6 +36,18 @@ Pointing your browser to ``http://127.0.0.1:8888/explore``
 will bring up a `GraphiQL`_ interface to EdgeDB. This interface can be
 used to try out queries and explore the GraphQL capabilities.
 
+
+Known Limitations
+=================
+
+- Due to the differences between EdgeQL and GraphQL syntax some
+  :eql:type:`enum <std::enum>` types cannot be properly reflected.
+
+- EdgeDB :eql:type:`tuples <std::tuple>` are not supported in GraphQL
+  reflection currently (due to the fact that GraphQL Lists are
+  supposed to be homogeneous and tuples are heterogeneous).
```

Comment:
Tuples are not lists, they are closer to objects, although lack ids. I suggest to drop the `currently (due .*)` part.

Reply:
Well, an unnamed tuple is just an ordered heterogeneous list. In fact, every tuple is defined by its element types and ordering and not by the optionally provided names. The point is that names are completely secondary in tuples unlike in object types. They are more like heterogeneous lists with a secondary optional indexing method by names.

Consider this: 
```
db> SELECT (1, 2) = (a := 1, b:= 2);
{true}
db> SELECT (b:= 2, a := 1) = (a := 1, b:= 2);
{false}
```

Reply:
> Well, an unnamed tuple is just an ordered heterogeneous list.

In languages that have "heterogeneous lists". Tuples are also frequently referred to as records. I'm not sure why we're having this debate though; for the user-facing documentation you just want to say in simple words what's supported and what's not without diving into too much detail/cs terminology.

Reply:
Maybe I'm overthinking it here. I'll drop the "due..." part.